packages/expo-random/android/src/main/java/expo/modules/random/RandomModule.kt

```diff
@@ -0,0 +1,27 @@
+package expo.modules.random
+
+import android.util.Base64
+
+import java.security.SecureRandom
+
+import com.facebook.react.bridge.Promise
+import com.facebook.react.bridge.ReactApplicationContext
+import com.facebook.react.bridge.ReactContextBaseJavaModule
+import com.facebook.react.bridge.ReactMethod
+
+class RandomModule(context: ReactApplicationContext) : ReactContextBaseJavaModule(context) {
+  private val secureRandom: SecureRandom by lazy { SecureRandom() }
```

Comment:
```suggestion
  private val secureRandom by lazy { SecureRandom() }
```
The type here is unnecessary, but is up to you 😉

Reply:
Type of this property is clear anyway, so let's go for it.